content/chainguard/chainguard-images/working-with-images/security-advisories/managing-advisories/index.md

```diff
@@ -0,0 +1,165 @@
+`-V` lists all advisories for a given CVE ID, across all packages.
+`-t` lists all advisories by their most recent event type.
+`-c` lists all advisories by detected component type.
+`--history` reports the full list of events for displayed advisories.
+Multiple flags can be combined together to produce a more granular search. For a full listing of available flags, you can run the `wolfictl advisory list -h` command in your terminal.
+
+For example, let’s say you want to find advisories for the `glibc` package, and we want to see the full history of these advisories. Note that the output of this command may change as vulnerability entries are added and updated over time.
+
+```shell
+wolfictl advisory list -p glibc
+```
+
+The following shows a small sample of output from this command at the time of execution.
+```
+…
+CGA-pjj8-8gh5-2qwh (CVE-2024-33599, GHSA-9gvm-vcgf-x5xw)   2024-05-14T11:00:49Z detected (glibc)
+                                                                 2024-05-15T04:39:53Z fixed (2.39-r5)
+      CGA-7fr2-v9gg-pg28 (CVE-2024-33600, GHSA-jv3g-6pg3-v9j8)   2024-05-14T11:00:51Z detected (glibc)
+                                                                 2024-05-15T04:40:16Z fixed (2.39-r5)
+      CGA-vjpp-g2r4-c8gc (CVE-2024-33601, GHSA-f4cf-2w52-c853)   2024-05-14T11:00:53Z detected (glibc)
+                                                                 2024-05-15T04:40:32Z fixed (2.39-r5)
+      CGA-wfxf-8642-f6f5 (CVE-2024-33602, GHSA-f4pv-q5f7-2h55)   2024-05-14T11:00:56Z detected (glibc)
+                                                                 2024-05-15T04:40:42Z fixed (2.39-r5)
+
+…
+```
+
+From this snapshot, we can get an idea of the timeline of a vulnerability’s remediation process. The first CVE displayed, CVE-2024-33599, was detected on 5/14/2023 and remediated within the next 24 hours. Similar results are shown for other advisories, including the versions of the package in which the vulnerability was remediated.
+
+We encourage you to experiment with these flags to see what information you can gather from your various searches.
+
+## Creating and Updating Advisories
+
+To begin creating and updating security advisories, we will need to navigate back to the package repository we cloned.
+
+```shell
+cd ../os 
+```
+
+Here, we will run the `wolfictl advisory create` command to begin the process of adding a new advisory. This command will respond with a few prompts in the terminal requesting input of information used to create the advisory.
+
+First, we will be asked to enter the package in which the CVE was found. For our demonstration we will use the `glibc` package. Then, we will be asked for the vulnerability ID that we wish to make an advisory for. This ID must be a CVE, CGA, GHSA, or Go vulnerability ID. Finally, we will be asked what the status for the advisory should be, from one of the following options:
+- detection (Under investigation)
```

Comment:
```suggestion
- `detection` (Under investigation)
```
I think this makes sense, right? Any time I'm referring to text that appears in a command or output I think it's helpful to have it formatted as code.

If you agree, can you add backticks to the remaining bullets?